openreview/tools.py

```diff
@@ -19,21 +19,25 @@
 from tqdm import tqdm
 from concurrent.futures import ThreadPoolExecutor
 
-def decision_to_venue(venue_id, decision_option):
+def decision_to_venue(venue_id, decision_option, accept_options=None):
     """
     Returns the venue for a submission based on its decision
 
     :param venue_id: venue's short name (i.e., ICLR 2022)
     :type venue_id: string
     :param decision_option: paper decision (i.e., Accept, Reject)
     :type decision_option: string
+    :param accept_options: accept decisions (i.e., [ Accept (Best Paper), Invite to Archive ])
+    :type accept_options: list
     """
     venue = venue_id
-    if 'Accept' in decision_option:
-        decision = decision_option.replace('Accept', '')
-        decision = re.sub(r'[()\W]+', '', decision)
+    decision = re.sub(r'[()\W]+', '', decision_option)
+    if not accept_options and 'Accept' in decision_option:
+        decision = decision.replace('Accept', '')
         if decision:
             venue += ' ' + decision.strip()
+    elif accept_options and decision_option in accept_options:
+        venue += ' ' + decision.strip()
```

Comment:
I think we always should replace `Accept` right? Some of the `accept_options` might have the word Accept and we should treat those the same way we used to.

Reply:
☝️

Reply:
It looks like with this approach, we are doing a combination of checking for 'Accept' and using `accept_decision_options`? 
For example, if I have an option `Accept (Oral)` in the decision_options, but not in accept_decision_options, it is still treated as an accept?
Not sure if this approach is right or wrong, but want to double check this is the desired behavior?

Reply:
Yup, basically we only check for "Accept" if `accept_decision_options` is empty. This is to make it backwards compatible and not let API 1 tests fail and have to make lots of changes

Reply:
That's correct and that is what is implemented now.

if accept_decision_options then check if decision is in option then marked as accept
else if 'Accept' is in decision then marked as accept

We need to reflect that in the documentation. Correct me if I'm wrong.

Reply:
ok! I thought this should be the behavior, but from what I see here we are always checking if the decision has 'Accept', regardless of if we have `accept_decision_options` or not. 

Just to be clear @melisabok: a decision of `Accept (Oral)` that was not specified in `accept_decision_options` should **not** be treated as an accept, assuming `accept_decision_options` is non-empty, correct?

Reply:
yes, accept_decision_options overwrites the "Accept" string rule.

you are right, the tools implementation should check the accept_decision_options first and then the string rule like in other parts of the code. Emilia could you change it?